Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+pywin32 = "*"
```

Comment:
I think we should set pywin32 version 306 here too. Maybe we could use one pipenv or requirements, otherwise we have to maintain dependencies in those two files. What do you think?

Reply:
Sobre el versionado, ahora mismo lo corrijo y añado la version 306.

Y lo que puedo hacer es crear un workflow en github que ejecute el comando que usa pipenv para generar el requirements.txt cada vez que se hace un push a la rama main. Y te parece bien ahora lo hago asi se pueden mantener ambos sistemas para que el usuario no necesite forzadamente usar pipenv.

Reply:
It's ok. Let's start simpler by having those two files.

Reply:
It's ok, let's start simpler by having those two files updated manually 👍

Reply:
Perfect, since I already had a similar workflow I had no problems adding it here, but if you want you can remove it.

Reply:
If everything is fine you can accept the pull request.